src/implementations/wrap/wrap.ts

```diff
@@ -16,6 +16,12 @@ interface Schema {
   $id: string;
 }
 
+export enum Output {
+  File = "File",
```

Comment:
you dont need to make this a string enum

Reply:
fixed